src/specfem2D/assemble_MPI.F90

```diff
@@ -129,9 +131,19 @@ subroutine assemble_MPI_scalar(array_val1,npoin_val1, &
         buffer_send_faces_scalar(ipoin,iinterface) = array_val4(iglob)
       enddo
 
+      ! electromagnetic
+      do idim = 1,NDIM
+        do i = 1, nibool_interfaces_electromagnetic(iinterface)
+          ipoin = ipoin + 1
+          iglob = ibool_interfaces_electromagnetic(i,iinterface)
+          buffer_send_faces_scalar(ipoin,iinterface) = array_val5(idim,iglob)
```

Comment:
the array `array_val5` has not been defined yet, so this will fail code compilation for the parallel version (WITH_MPI) - please fix this and re-submit the PR.